plugins/services/src/js/components/ServiceStatusProgressBar.js

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from "react";
+import { Tooltip } from "reactjs-components";
+
+import StringUtil from "#SRC/js/utils/StringUtil";
+import ProgressBar from "#SRC/js/components/ProgressBar";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceTree from "../structs/ServiceTree";
+import ServiceStatus from "../constants/ServiceStatus";
+
+class ServiceStatusProgressBar extends React.Component {
+  getTooltipContent() {
+    const { service } = this.props;
+    const { tasksRunning } = service.getTasksSummary();
+    const instancesTotal = service.getInstancesCount();
+
+    return (
+      <div className="tooltip-line-item">
+        <span>
+          {`${tasksRunning} ${StringUtil.pluralize("instance", tasksRunning)} running out of ${instancesTotal} `}
+        </span>
+      </div>
+    );
+  }
+
+  getProgressBar(data) {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+
+    if (tasksRunning === instancesCount) {
+      return (
+        <ProgressBar
+          className="status-bar--large"
+          data={data}
+          total={instancesCount}
+        />
+      );
+    }
+
+    return (
+      <Tooltip interactive={true} content={this.getTooltipContent()}>
+        <ProgressBar
+          className="status-bar--large"
+          data={data}
+          total={instancesCount}
+        />
+      </Tooltip>
+    );
+  }
+
+  render() {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const serviceStatus = service.getStatus();
```

Comment:
Why `getStatus` and not `getServiceStatus` like `StatusIcon`?

Reply:
`getStatus` uses `getServiceStatus` but only returns the display name which is what I need.

Reply:
I left a comment about this over here https://github.com/dcos/dcos-ui/pull/2475#discussion_r148685344